test-cases/tests/alerts/c12-verify-no-critical-alerts-have-fired-on-cluster-during-lifes.md

```diff
@@ -0,0 +1,28 @@
+---
+targets:
+  - 2.6.0
```

Comment:
Not sure about this tag. My understanding was that if target is specified then the test case will on be verified for given release. This is not a one-off test case.

Reply:
Yes, I think you are right, will update PR